dagmc_stats/DagmcStats.py

```diff
@@ -0,0 +1,253 @@
+        for tri in tris:
+            side_lengths = list(get_tri_side_length(tri).values())
+            s = .5*(sum(side_lengths))
+            if area:
+                # sqrt(s(s - a)(s - b)(s - c)), where s = (a + b + c)/2
+                t_a = np.sqrt(s * np.prod(s - side_lengths))
+                area.append(t_a)
+            if aspect_ratio:
+                top = np.prod(side_lengths)
+                bottom = 8*np.prod(s-side_lengths)
+                t_a_r.append(top/bottom)
+        if area:
+            df_tri['tri_area'] = area
+        if aspect_ratio:
+            df_tri['t_a_r'] = t_a_r
+            
+        """if area:
+            area = []
+            for tri in tris:
+                side_lengths = list(get_tri_side_length(tri).values())
+                s = .5*(sum(side_lengths))
+                # sqrt(s(s - a)(s - b)(s - c)), where s = (a + b + c)/2
+                t_a = np.sqrt(s * np.prod(s - side_lengths))
+                area.append(t_a)
+            df_tri['tri_area'] = area
+        if aspect_ratio:
+            t_a_r = []
+            for tri in tris:
+                side_lengths = list(get_tri_side_length(tri).values())
+                s = .5*(sum(side_lengths))
+                top = np.prod(side_lengths)
+                bottom = 8*np.prod(s-side_lengths)
+                t_a_r.append(top/bottom)
+            df_tri['t_a_r'] = t_a_r"""
+        return
+        
+
+    def get_tris(self, meshset):
+        """
+        Get triangles of a volume if geom_dim is 3
+        Get triangles of a surface if geom_dim is 2
+        Else get all the triangles
+
+        inputs
+        ------
+        meshset : set of entities that are used to populate data. Default value
+        is None and data of the whole geometry will be populated.
+
+        outputs
+        -------
+        tris : a list of triangle entities
+        """
+        #Todo: should be here or in init()?
+        dim = _my_moab_core.tag_get_data(dagmc_tags['geom_dim'], meshset)[0][0]
+        # get triangles of a volume
+        if dim == 3:
+            entities = _my_moab_core.create_meshset()
+            for surface in _my_moab_core.get_child_meshsets(root_set):
+                _my_moab_core.add_entities(
+                    entities, _my_moab_core.get_entities_by_type(surface, types.MBTRI))
+            tris = _my_moab_core.get_entities_by_type(entities, types.MBTRI)
+        # get triangles of a surface
+        elif dim == 2:
+            entities = _my_moab_core.create_meshset()
+            my_moab_core.add_entities(
+                entities, _my_moab_core.get_entities_by_type(root_set, types.MBTRI))
+            tris = _my_moab_core.get_entities_by_type(entities, types.MBTRI)
+        else:
+            # get all the triangles
+            tris = _my_moab_core.get_entities_by_type(root_set, types.MBTRI)
+        return tris
+   
+   
+    def get_tri_side_length(self, tri):
+        """
+        Get side lengths of triangle
```

Comment:
```suggestion
        """Get side lengths of triangle
```

Reply:
Is this PEP8?  I find the original more aesthetically pleasing, but defer to PEP8.